docs/flowcollector-flows-netobserv-io-v1alpha1.adoc

```diff
@@ -0,0 +1,1248 @@
+--
+
+Type::
+  `object`
+
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `enable`
+| `boolean`
+| enable the console plugin port-to-service name translation
+
+| `portNames`
+| `object (string)`
+| portNames defines additional port names to use in the console. Example: portNames: {"3100": "loki"}
+
+|===
+== .spec.consolePlugin.quickFilters
+Description::
++
+--
+quickFilters configures quick filter presets for the Console plugin
+--
+
+Type::
+  `array`
+
+
+
+
+== .spec.consolePlugin.quickFilters[]
+Description::
++
+--
+QuickFilter defines preset configuration for Console's quick filters
+--
+
+Type::
+  `object`
+
+Required::
+  - `filter`
+  - `name`
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `default`
+| `boolean`
+| default defines whether this filter should be active by default or not
+
+| `filter`
+| `object (string)`
+| filter is a set of keys and values to be set when this filter is selected. Each key can relate to a list of values using a coma-separated string. Example: filter: {"src_namespace": "namespace1,namespace2"}
+
+| `name`
+| `string`
+| name of the filter, that will be displayed in Console
+
+|===
+== .spec.consolePlugin.resources
+Description::
++
+--
+resources, in terms of compute resources, required by this container. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+--
+
+Type::
+  `object`
+
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `limits`
+| `integer-or-string`
+| Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+
+| `requests`
+| `integer-or-string`
+| Requests describes the minimum amount of compute resources required. If Requests is omitted for a container, it defaults to Limits if that is explicitly specified, otherwise to an implementation-defined value. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+
+|===
+== .spec.exporters
+Description::
++
+--
+exporters defines additional optional exporters for custom consumption or storage. This is an experimental feature. Currently, only KAFKA exporter is available.
+--
+
+Type::
+  `array`
+
+
+
+
+== .spec.exporters[]
+Description::
++
+--
+FlowCollectorExporter defines an additional exporter to send enriched flows to
+--
+
+Type::
+  `object`
+
+Required::
+  - `type`
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `kafka`
+| `object`
+| kafka describes the kafka configuration (address, topic...) to send enriched flows to.
+
+| `type`
+| `string`
+| type selects the type of exporters. Only "KAFKA" is available at the moment.
+
+|===
+== .spec.exporters[].kafka
+Description::
++
+--
+kafka describes the kafka configuration (address, topic...) to send enriched flows to.
```

Comment:
Should this be "Describes the Kafka configuration..." and can we change the "(address, topic...)", so its like this: "Describes the Kafka configuration, such as address or topic, to send enriched flows."

Reply:
=> https://github.com/netobserv/network-observability-operator/pull/242

(FYI, it's kind of a norm in golang's documentation to start commenting a field with it's own name, and since it's used as the source of the doc, it ends up in the asciidoc.)